http/http-advanced/src/main/java/io/quarkus/ts/http/advanced/HeadersResource.java

```diff
@@ -13,6 +15,13 @@ public String headers() {
         return "ok";
     }
 
+    @GET
+    @Path("/no-accept")
+    @Produces(MediaType.TEXT_PLAIN)
+    public CustomHeaderResponse noAcceptheaders() {
+        return new CustomHeaderResponse("ok");
```

Comment:
It would be nice to have same return String here as it's in reactive counter-part. You have there `"ok headers"`